app/routes/closure.js

```diff
@@ -0,0 +1,106 @@
+const fs = require('fs')
+const { closureAdmin } = require('../auth/permissions')
+const schema = require('./schemas/closure')
+const bulkSchema = require('./schemas/bulk-closure')
+const parsedSchema = require('./schemas/parsed-closure')
+const { post } = require('../api')
+
+module.exports = [{
+  method: 'GET',
+  path: '/closure',
+  options: {
+    auth: { scope: [closureAdmin] },
+    handler: async (_request, h) => {
+      return h.view('closure')
+    }
+  }
+},
+{
+  method: 'GET',
+  path: '/bulk-closure',
```

Comment:
Same comment as in processing that these endpoints could be more REST compliant.  Though appreciate the others in this repo probably lack that standardisation